.gitlab/internal_kubernetes_deploy.yml

```diff
@@ -28,4 +28,4 @@ internal_kubernetes_deploy_experimental:
   script:
     - source /root/.bashrc
     - export GITLAB_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.gitlab_pipelines_scheduler_token --with-decryption --query "Parameter.Value" --out text)
-    - inv pipeline.trigger-child-pipeline --project-name "DataDog/k8s-datadog-agent-ops" --git-ref "main" --variables "OPTION_AUTOMATIC_ROLLOUT,EXPLICIT_WORKFLOWS,OPTION_PRE_SCRIPT,FILTER,SKIP_PLAN_CHECK" --follow false
```

Comment:
Is the `FILTER` removal intentional? It doesn't seem related to the described change

Reply:
Yea, the variable was removed in the linked PR, but I did not see it in the passed in variable list on this line.

Reply:
Thanks for clearing it out :)